src/utils/navigation.ts

```diff
@@ -25,6 +25,20 @@ export function keyword(node: ts.BreakOrContinueStatement | ts.ThrowStatement |
   return node.getFirstToken();
 }
 
+export function isAssignment(node: ts.Node): node is ts.BinaryExpression {
+  return (
+    node.kind === ts.SyntaxKind.BinaryExpression &&
+    (node as ts.BinaryExpression).operatorToken.kind === ts.SyntaxKind.EqualsToken
+  );
+}
+
+export function retrievePureIdentifier(node: ts.Node): ts.Identifier | undefined {
```

Comment:
@inverno The name of the function doesn't reflect the functionality. And I would not put such narrow function to a shared module. Could we generalise it?